src/orbit/lib/arrays.js

```diff
@@ -24,4 +24,16 @@ function some(array, predicate) {
   return false;
 }
 
-export { every, some };
+function firstResult(array, valueFn) {
```

Comment:
This is like a `reduce` with an early return. Could also reuse `some` while modifying a local variable. I feel like I might be missing something obvious!